server/routes/objects.js

```diff
@@ -16,8 +16,13 @@ import {
   file_stats,
   file_uploaded,
 } from '../Database_queries/queries.js';
-import { sockets, removeSocket } from '../SocketManager/socketmanager.js';
-import { v4 as uuidv4 } from 'uuid';
+import { logger, log } from '../logger.js';
+import {
```

Comment:
The import statement for `emitMessage` is correct, but ensure that the function is properly defined in the `socketmanager.js` file. If it is not defined, it will lead to runtime errors.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*